doc/templates/uno/component/navbar.scss

```diff
@@ -14,7 +14,7 @@
  */
 
 .visibility-opacity-transition {
-  @include media-breakpoint-up(lg) {
+  @media (min-width: 1112px) {
```

Comment:
Surprising that there is not way to share this value `1112px` from a centralized place (a variable) like the colors.

Reply:
@carldebilly  I mean like saving the value in a variable 

```scss
$min-width : 1112px
```

Then just reusing this value everywhere. I read this is not possible with Media Query because of their nature.